streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImpl.java

```diff
@@ -74,6 +74,7 @@
     private final OffsetCheckpoint checkpointFile;
     private final Map<TopicPartition, Long> checkpointFileCache;
 
+    @SuppressWarnings("deprecation") // TODO: remove in follow up PR when `RETRIES` is removed
```

Comment:
Thanks for adding the comment!

Reply:
Yes :) Added it as a reminder only first. Was expecting to get review comment anyway.

Reply:
Addressed via #9047